composer.json

```
@@ -11,15 +11,14 @@
         }
     ],
     "require": {
-        "php": "^7",
-        "twig/twig": "^2",
+        "twig/twig": "^2.7 || ^3",
         "zicht/classes": "^2"
     },
     "require-dev": {
         "phpunit/phpunit": "^4"
     },
     "autoload": {
-        "psr-4": { "": ["src/"] }
+        "psr-4": { "Zicht\\Twig\\Extension\\": ["src/"] }
```

Dit moet zijn:

```suggestion
        "psr-4": { "Zicht\\Twig\\Extension\\": ["src/Zicht/Twig/Extension/"] }
```

OF je kunt alle bestanden in `src/Zicht/Twig/Extension/` verplaatsen naar `src/`, dan werkt bovenstaande PSR-4 loader wel

---

:+1: